src/Object/Assign.ts

```diff
@@ -12,19 +12,26 @@ import {Depth} from './_Internal'
 /**
 @hidden
 */
-type __Assign<O extends object, Os extends List<object>, depth extends Depth, I extends Iteration = IterationOf<'0'>> = {
```

Comment:
This does not improve readability for the end user, it will get reformatted for the end user. The docs should reflect the arguments instead. Please keep it on one line :)